support/cas-server-support-themes-collection/src/main/resources/templates/twbs/fragments/scripts.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript" th:src="@{${#themes.code('webjars.bootstrapmin.js')}}"></script>
+<script type="text/javascript" th:src="@{${#themes.code('cas.standard.js.file')}}"></script>
```

Comment:
It was necessary to override this file to control when the files are loaded. I also wanted to remove the reference to Material for this theme since the library is unused and large in file size.

Reply:
Nvm, found the new references and removed scripts.html